src/test/java/org/dataloader/DataLoaderValueCacheTest.java

```diff
@@ -430,9 +443,9 @@ public CompletableFuture<List<Object>> setValues(List<String> keys, List<Object>
         };
         customValueCache.asMap().put("a", "cachedA");
 
-        List<List<String>> loadCalls = new ArrayList<>();
+        List<Collection<String>> loadCalls = new ArrayList<>();
         DataLoaderOptions options = newOptions().setValueCache(customValueCache).setCachingEnabled(true).setBatchingEnabled(false);
-        DataLoader<String, String> identityLoader = idLoader(options, loadCalls);
+        DataLoader<String, String> identityLoader = factory.idLoader(options, loadCalls);
 
         CompletableFuture<String> fA = identityLoader.load("a");
         CompletableFuture<String> fB = identityLoader.load("b");
```

Comment:
Where is the new test that shows the bug happening.  I cant see it for all the other changes (and I have merged #171 )

Reply:
My bad for not explaining this. There aren't any "new" tests, as such - but we are now testing the existing tests here against the `*PublisherDataLoader` factories in addition to the standard ones (see https://github.com/graphql-java/java-dataloader/pull/172/commits/9a20334effd34fc1dd31c9592b3ac278c6fbc464#r1901980139).